docs/_static/policy/v1.yaml

```diff
@@ -0,0 +1,133 @@
+          It is a space separated list of scopes.
+        required: true
+        schema:
+          type: string
+    get:
+      summary: Returns a presentation definition for the given DID and scope.
+      description: |
+        The DID is used for tenant selection. Not all tenants will probably support the same scopes.
+        The scope is used as selection criteria for the presentation definition.
+        It could be the case that the presentation definition is not found.
+        In that case the response will be 201 with an empty body.
+      operationId: "presentationDefinition"
+      tags:
+        - policy
+      responses:
+        "200":
+          description: |
+            DID has been found and the scope is supported.
+            If the scope is supported but no presentation definition is required, the response will be 200 with a presentation definition without any input descriptors.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PresentationDefinition'
+        "201":
+          description: The DID is known but the presented scope is not supported.
+        "404":
+          description: DID is not known to the policy backend.
+  /{did}/authorize:
+    parameters:
+      - name: did
+        in: path
+        description: URLEncoded DID. The DID is used for tenant selection.
+        required: true
+        example: did:web:example.com:1
+        schema:
+          type: string
+    post:
+      summary: Authorize a resource request.
+      description: |
+        When an access token is used to request a resource, the resource server needs to know if the access token grants access to the requested resource.
+        The resource server will send a request to the policy backend to check if the access token grants access to the requested resource.
+
+      operationId: "authorize"
+      tags:
+        - policy
+      requestBody:
+        description: Required params for policy backend to make an informed decision.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AuthorizeRequest'
+      responses:
+        "200":
+          description: A response that indicates if the access token grants access to the requested resource.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AuthorizeResponse'
+        "404":
+          description: DID is not known to the policy backend.
+components:
+  schemas:
+    AuthorizeRequest:
+      description: |
+        The request contains all params involved with the request.
+        It might be the case that the caller mapped credential fields to additional params.
+      type: object
+      required:
+        - client_id
+        - scope
+        - request_url
+        - request_method
+        - presentation_submission
+        - vps
+      properties:
```

Comment:
missing the `input_desciptor_constraint_id_map` from the introspection endpoint? Could probably use a simpler name here.

Reply:
Then the entire id to value mapping would have to be standardised. I think it's too early for a decision on that. With the VPs included, a backend can make a decision.

Reply:
That'll be an incremental change. We can then also process feedback in the same change.